code-challenges.test.js

```diff
@@ -15,6 +15,16 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("divByThree", () => {
+  it("takes a number and decides if it is evenly divisble by three or not", () => {
+    expect(divByThree(15)).toEqual("15 is divisible by three")
+    expect(divByThree(0)).toEqual("0 is divisible by three")
+    expect(divByThree(-7)).toEqual("-7 is not divisible by three")
+  })
+})
+
+//Ran $ yarn jest and got 1 failed. error Command failed with exit code 1.
```

Comment:
The purpose of the test variables is to use them in your test as the argument to your function.